bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -67,14 +67,15 @@ <h1>{{ self.page_title() }}</h1>
     ] -%}
     {% include promos|random() %}
   {% else %}
-    {% call hero(
-      title='Firefox products are designed to protect your privacy',
-      class='privacy-promise-hero mzp-has-image mzp-t-dark mzp-t-firefox',
+    {% call split(
       image_url='img/firefox/privacy/promise/privacy-hero.png',
       include_highres_image=True,
-      include_cta=True,
-      heading_level=2
+      media_after=True,
+      media_class='mzp-l-split-h-center',
+      block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md mzp-t-dark mzp-t-firefox',
+      theme_class=''
     ) %}
+      <h2 class="mzp-u-title-lg">Firefox products are designed to protect your privacy</h2>
```

Comment:
Looks like this isn't inheriting the correct font from the mzp-t-firefox theme class. It should be Metropolis instead of Mozilla slab:

![image](https://user-images.githubusercontent.com/400117/130627169-fb2e287b-a672-46f6-bbb0-25163e125ce9.png)

I think this is happening because the page is a mix of different brand themes for both Mozilla and Firefox. You may need to explicitly style this heading using `@include font-firefox;`